Description: Fix FTBFS due to wrong file list in po/POTFILES.in
Author: Andrea Colangelo <warp10@ubuntu.com>

--- gbemol-0.3.2.orig/po/POTFILES.in
+++ gbemol-0.3.2/po/POTFILES.in
@@ -1,3 +1,8 @@
 # List of source files containing translatable strings.
-
-src/main.c
+src/bacon-volume.c
+src/gbemol-app.c
+src/gbemol-info.c
+src/gbemol-library.c
+src/gbemol-main.c
+src/gbemol-playlist.c
+src/gbemol-preferences.c
